worksheet_A/TerminalHacking.cpp

```diff
@@ -39,8 +62,57 @@ int main()
 		std::cout << word << std::endl;
 	}
 
+	
 	// TODO: implement the rest of the game
+	
+	//Only runs loop while the player has lives left
+	while (lives > 0)
+	{
+		correctguess = false;
+		while (correctguess == false)
+		
+		{	
+			// Allows the player to enter an anwser
+			std::cout << "Please Enter Your Guess in CAPS\n";
+			
+			std::cin >> input;
+
+			// Checks all of the words in the list against the player's guess
+			for each (std::string word in options)
+			
+		
+			{	// Checks that the player's word is actually in the list
+				if (input == word)
+				{
+					correctguess = true;
+				}
+				
+			}
+		}
 
-    return 0;
+		// Gets the likeness of value of the player's guess against the secret word, by calling the created function
+		LikenessScore = getlikeness(input, secret);
+		std::cout << "Likeness score was = " << LikenessScore << "/" << wordlength << std::endl;
+		
+		// Checks if the inputted guess was correct
+		if (LikenessScore == wordlength)
+		{
+			std::cout << "You Win!\n";
+			return 0;
+		}
+		
+		
+		else
+		{
+			// Removes a life if the correct anwser is not given by the player
+			lives--;
+			std::cout << "Incorrect Guesses left = " << lives << "/4\n";
```

Comment:
When using `std::cout`, generally better to use `std::endl` rather than `"\n"` -- the latter might have portability issues.